interfaces/builtin/camera.go

```diff
@@ -47,17 +60,89 @@ const cameraConnectedPlugAppArmor = `
 /sys/devices/pci**/usb*/**/video4linux/** r,
 `
 
-var cameraConnectedPlugUDev = []string{`KERNEL=="video[0-9]*"`}
+const cameraConnectedPlugAppArmorHotplug = `
+###PATH### rw,
+
+# Allow listing of all devices, however only the designated camera can discovered
+/sys/bus/usb/devices/ r,
+###DEVPATH### r,
+/run/udev/data/c81:###MINOR### r, # video4linux (/dev/video*, etc)
+/sys/class/video4linux/ r,
+`
+
+var cameraConnectedPlugUDev = []string{`KERNEL=="###KERNEL###"`}
+
+// Name of the serial-port interface.
+func (iface *cameraInterface) Name() string {
+	return "camera"
+}
+
+func (iface *cameraInterface) String() string {
+	return iface.Name()
+}
+
+func (iface *cameraInterface) StaticInfo() interfaces.StaticInfo {
```

Comment:
Can you please embed commonInterface and remove the methods that don't deviate from the generic implementation there?